easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.0-GCCcore-6.4.0.eb

```diff
@@ -26,7 +26,7 @@ builddependencies = [
 dependencies = [
     ('expat', '2.2.5'),
     ('freetype', '2.9'),
-    ('LibUUID', '1.0.3'),
+    ('util-linux', '2.32.1'),
```

Comment:
@schiotz This should be `'2.31.1'` (since this version is already used as dependency in easyconfigs using a toolchain from the `2018a` generation), tests will most likely fail without fixing this.

Reply:
@boegel Done.